rxdogtag/src/test/java/com/uber/anotherpackage/DogTagObserverTest.java

```diff
@@ -120,6 +120,28 @@ public void observerWithErrorHandling_ignoredByRxDogTag() {
         });
   }
 
+  @Test
+  public void nullOriginErrorMessage_replacedWithEmptyString() {
+    Exception originalError = new IllegalStateException();
+    int expectedLineNumber = subscribeError(Observable.error(originalError));
+
+    Throwable e = errorsRule.take();
+    assertThat(e).isInstanceOf(OnErrorNotImplementedException.class);
+    // Original error message was null. Now replaced by empty string
```

Comment:
I wonder if we should put something more descriptive here, like `"<no error message>"`.